code/datums/diseases/critical.dm

```diff
@@ -33,6 +33,7 @@
 	cures = list("salglu_solution", "syndicate_nanites", "stimulative_agent")
 	cure_chance = 10
 	stage_prob = 6
+	visibility_flags = HIDDEN_PANDEMIC
```

Comment:
Put this on `/datum/disease/critical`, none of the critical viruses should be seen